### DISCUSSION TITLE
Question - how to add custom environment

### DISCUSSION THREAD

**Discussion by AlekMosingiewicz** [pimcore/pimcore#9407] [Q&A / Support]
We need to add a custom environment ("dv") to our project. We added appropriate config file to appropriate location ("/config/packages/dv/config.yaml"), with all the required imports, we still get

"The service "Pimcore\Log\Handler\ApplicationLoggerDb" has a dependency on a non-existent service "monolog.processor.psr_log_message"."

error. As far as I understand, this is because the services.yaml file is not loaded, and this in turn because config file for specific environment is not loaded. What are we doing wrong?

**Comment by dvesh3** [comment]
@AlekMosingiewicz if you are using Pimcore X version < 10.0.2 then you will face this issue since `MonologPsrLogMessageProcessorPass` (which redefines the service) was removed by mistake on 10.0 and this was reverted in 10.0.2 please see https://github.com/pimcore/pimcore/pull/9254
